cohttp-lwt-unix/test/test_sanity.ml

```diff
@@ -84,6 +84,7 @@ let check_logs test () =
 
 let ts =
   Cohttp_lwt_unix_test.test_server_s server (fun uri ->
+      let uri = Uri.with_host uri (Some "localhost") in
```

Comment:
This was not a problem before, why does it need change?

Reply:
the uri is something like `http://0.0.0.0:6394`. Listening on `0.0.0.0` is fine, but connecting to `0.0.0.0` / `ANY_ADDRESS` is a linuxism. I'm on OpenBSD.

Reply:
Ok but then let’s fix it here: https://github.com/mirage/ocaml-cohttp/blob/7aa9a33ab1a9fa9fe767b5fe259251436e41fd02/test_helpers/cohttp_lwt_unix_test/src/cohttp_lwt_unix_test.ml#L32

Reply:
done.